tests/conftest.py

```diff
@@ -113,6 +118,17 @@ def identity():
     return i
 
 
+@pytest.fixture(scope="module")
+def superuser_identity():
+    """Super user identity to interact with the services."""
+    i = Identity(2)
+    i.provides.add(UserNeed(2))
+    i.provides.add(any_user)
+    i.provides.add(system_process)
```

Comment:
My confusion about who can read the record is essentially this fixture. The superuser can read because they have the `system_process` need not because of the `superuser_access`